src/main/java/com/training/test/controller/RestaurantController.java

```diff
@@ -0,0 +1,55 @@
+package com.training.test.controller;
+
+import com.training.test.entity.RestaurantDetails;
+import com.training.test.model.RestroDetailsRequest;
+import com.training.test.service.LoginService;
+import com.training.test.service.RestroService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/restro")
+public class RestaurantController {
+
+    //    @Autowired
```

Comment:
Remove commented code. It is a bad practice to commit unused code.

Reply:
removed